ipa-core/src/net/server/handlers/mod.rs

```diff
@@ -17,6 +17,8 @@ pub fn mpc_router(transport: MpcHttpTransport) -> Router {
 pub fn shard_router(transport: ShardHttpTransport) -> Router {
     echo::router().nest(
         http_serde::query::BASE_AXUM_PATH,
-        Router::new().merge(query::s2s_router(transport)),
+        Router::new()
+            .merge(query::c2s_router(&transport))
```

Comment:
I would call it something like `shard_query_router` to match the one in mpc

Reply:
I don't mind either. Regardless of scheme we choose here, the naming will be inconsistent. We either deviate from `s2s` or from MPC counterpart. Let me know which one you prefer

Reply:
ok, we don't need this anymore